docs/README.md

```diff
@@ -1,5 +1,10 @@
+<img src="https://raw.githubusercontent.com/CodeShady/redownload/main/logo.png" />
```

Comment:
Unless I decide to have a full-on logo made for the project (which is really not on the radar right now), I'd rather not put one in the README.